WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -170,6 +166,17 @@ - (void)viewWillDisappear:(BOOL)animated
     self.resultsController.delegate = nil;
 }
 
+- (void)didRotateFromInterfaceOrientation:(UIInterfaceOrientation)fromInterfaceOrientation
```

Comment:
Strange. All calculations should have been applied by the time `willAnimateRotationToInterfaceOrientation: duration:` is called. I tested the change before suggesting it and it seemed to work fine.  
On the other hand, the same is not true of `willRotateToInterfaceOrientation: duration:`. Could it have been that method you tried?